docs/apis-clients/go-client/get-started.md

```diff
@@ -135,7 +135,7 @@ The broker stores the process under its BPMN process id and assigns a version.
 ```go
 	// After the client is created
 	ctx := context.Background()
-	response, err := client.NewDeployProcessCommand().AddResourceFile("order-process.bpmn").Send(ctx)
+	response, err := client.NewDeployResourceCommand().AddResourceFile("order-process.bpmn").Send(ctx)
```

Comment:
:thought_balloon: Do we need to adjust the output too?

The output of the command is written in the docs under the code examples. If the output is changed then we should adjust it on this page too.

Reply:
👍 Good point. 

I did not check this. I went with a search tool through all matches and made changes where I thought they were necessary. I missed this one, but I totally agree that output should be updated as well.

Reply:
This took a bit more time because we discovered a problem in the v8 client (see https://github.com/camunda/zeebe/pull/9062).